Makefile

```diff
@@ -69,6 +69,7 @@ run-and-test: start test stop
 test: .env
 	@echo "\n========== STAGE: test ✅ ==========\n"
 	npm run test:convert
+	npm run pre:test
```

Comment:
You don't need to call this specifically if the script was `pretest` instead of `pre:test` (my mistake when I was typing in slack).